lib/Runtime/Library/SparseArraySegment.inl

```diff
@@ -241,7 +241,7 @@ namespace Js
             }
             for (i = (start + step-1)/step; i < (size/step); i++)
             {
-                ((Var*)(this->elements))[i] = fill;
+                ((Var*)(this->elements))[i] = fill; // swb: no write barrier, set to non-GC pointer
```

Comment:
We can... I remember I had that but deleted, worried it might be abused. We can think more about the best solution for cases like this. (However the cast here is mainly because "elements" are of "T[]" type, so it needs to cast to "Var[]" before assigning to "Var fill" value.)